app/views/ingredient_categories/index.html.erb

```diff
@@ -0,0 +1,41 @@
+<div class="column">
+  <h3 class="title has-text-white-ter has-text-right">
+    Ingredient categories
+  </h3>
+  <hr class="login-hr">
+  <p class="subtitle has-text-white-ter has-text-right">
+    We already have <%= IngredientCategory.count %> ingredient categories!
```

Comment:
Take a look at this https://github.com/Lohenrex/mogu-mogu/pull/13#pullrequestreview-865595515 👀

Reply:
Already fixed in the latest commit!